.github/workflows/check-stackblitz-templates.yml

```
@@ -1,28 +1,42 @@
 name: Stackblitz templates
-run-name: Playwright stackblitz templates 🚀
-on: 
-  workflow_dispatch:
-  schedule:
-    - cron: '59 23 * * *'
+
+
+run-name: Playwright is testing stackblitz templates 🚀
```

```suggestion
run-name: Playwright stackblitz templates 🚀
```
let's stay with the name for consistency